Apps/frontend/src/components/DistributeOffsetButton.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  import { NUM_CHANNELS } from "../const.js";
+  export let activeChannelCount;
+
+  const dispatch = createEventDispatcher();
+  const handleClick = async () => {
+    dispatch("distributeOffset", { offset: calculateOffset() });
```

Comment:
All events are conected via "`-`" (here: `distribute-offset`)

Reply:
They have been camel case so for, why are we changing that?

Reply:
Looked into it, neither of us in wrong ;)
In some places we use lowerCamelCase, in some we separate with "`-`". The svelte-given events actually use neither (see `on:mousedown`...)
I would have prefered a consistent way, but we should probably look into this in a future refactoring issue.

Reply:
Oh I see, yes would fit together with the test by id vs cy-data refactoring.